source/driver/Driver.cpp

```diff
@@ -143,6 +158,18 @@ void Driver::addStandardArgs() {
 
     cmdLine.setPositional(
         [this](string_view fileName) {
+            if (!options.excludeExtDone) {
```

Comment:
This boolean flag is kind of clunky, and furthermore relies on additional --exclude-ext args never coming after a positional argument, which nothing will currently prevent. I don't think you need it though; instead of storing the exclude-ext args in a vector, make that option take a callback that adds the values directly to the set instead and then you won't need a separate data structure.

Reply:
I didn't realize you could mix positional arguments and flags, because that never happens on standard Linux tools, but after you mentioned this, I realized you are right. Each '-f' file usually contains both filenames and incdir/defines/libdir flags, so the end result is that they are really mixed.

I'll modify the implementation to add members directly to the set, similar to the std::map code I had before.

Reply:
as you said, the callback method was the simplest.